portfolio/src/main/webapp/script.js

```diff
@@ -17,7 +17,10 @@
  */
 function addRandomFact() {
   const facts =
-     ['I am 20 years old','I have three siblings','I like to make music','I like to code'];
+     ['I am 20 years old.','I have three siblings.','I like to make music.',
+     'I like to code.', 'I like to play guitar.','I like to create.',
+     'I like tech.', 'I like to cook, but Im still learning.', 'I like seafood.', 'I like tacos.',
```

Comment:
Your 'I like to cook, but Im still learning.' Is missing a ' between the Im. You can add this by escaping the ' or changing your outside quotations to double quotes, so you can use single quotes in your sentences.

Reply:
You can escape the single quote within single quotes by using a backslash.

'Hi, I\'m Justin'